http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -168,7 +251,8 @@ object persistentDiskQuery extends TableQuery(new PersistentDiskTable(_)) {
       disk.size,
       disk.diskType,
       disk.blockSize,
-      disk.formattedBy
+      disk.formattedBy,
+      None //TODO: update this one PersistentDisk model is updated
```

Comment:
Are you planning to do this? Or should we leave this out of the `PersistentDisk` model?

Reply:
I'm not sure yet..but will revisit in the follow up PR